tests/model_fields/test_decimalfield.py

```diff
@@ -14,7 +14,8 @@ def test_to_python(self):
         f = models.DecimalField(max_digits=4, decimal_places=2)
         self.assertEqual(f.to_python(3), Decimal('3'))
         self.assertEqual(f.to_python('3.14'), Decimal('3.14'))
-        with self.assertRaises(ValidationError):
+        msg = "%r value must be a decimal number."
```

Comment:
No need to interpolate `'abc'` in the message -- this might be done in some existing tests for Python 2 compatibility to account for u'' prefixes on Python 2 but could be removed now.